src/odemis/util/test/util_test.py

```diff
@@ -543,6 +543,20 @@ def test_expand_rect(self):
                                                        "expanded rectangle %s" % (expanded_rect, expected_rect))
 
 
+class GetMinMaxTupleTestCase(unittest.TestCase):
+
+    def test_minmax(self):
+        """Test getting the minimum and maximum values from a list of tuples"""
```

Comment:
can you add a test case that checks what happens when you give the function negative values?